ui/src/app/components/with-error-modal-wrapper.tsx

```diff
@@ -76,7 +76,7 @@ export const withProfileErrorWrapper = (WrappedComponent) => {
     </React.Fragment>
   );
 
-  const ProfileErrorWrapper = ({ showErrorModal, ...props }) => {
+  return ({ showErrorModal, ...props }) => {
     return (
       <WrappedComponent
```

Comment:
Love that you removed the need for the assignment. I wonder why this isn't:
```javascript
return ({ showErrorModal, ...props }) => <WrappedComponent ... />
```

Reply:
good catch, thanks